modules/models/src/main/java/org/eclipse/xpanse/modules/models/response/ResultType.java

```diff
@@ -72,7 +72,8 @@ public enum ResultType {
     SERVICE_ORDER_NOT_FOUND("Service Order Not Found"),
     SERVICE_PRICE_CALCULATION_FAILED("Service Price Calculation Failed"),
     INVALID_GIT_REPO_DETAILS("Invalid Git Repo Details"),
-    FILE_LOCKED("File Locked");
+    FILE_LOCKED("File Locked"),
+    SERVICE_CONFIGURATION_NOT_VALID("Service Configuration Not Valid");
```

Comment:
It is better to rename it to 'INVALID_SERVICE_CONFIGURATION' for maintaining a consistent style with other custom exceptions